databuilder/databuilder/extractor/glue_extractor.py

```diff
@@ -21,11 +21,13 @@ class GlueExtractor(Extractor):
     FILTER_KEY = 'filters'
     MAX_RESULTS_KEY = 'max_results'
     RESOURCE_SHARE_TYPE = 'resource_share_type'
+    REGION_NAME = "region"
     DEFAULT_CONFIG = ConfigFactory.from_dict({
         CLUSTER_KEY: 'gold',
         FILTER_KEY: None,
         MAX_RESULTS_KEY: 500,
-        RESOURCE_SHARE_TYPE: "ALL"
+        RESOURCE_SHARE_TYPE: "ALL",
+        REGION_NAME: None
```

Comment:
If there is no default we don't really need this in the config

Reply:
I agree about `REGION_NAME_KEY`. However, if we wont put it as default, tests gives errors `pyhocon.exceptions.ConfigMissingException: 'No configuration setting found for key region' `.
I think it should be there as `FILTER_KEY`.